packages/simcore-sdk/tests/unit/test_node_ports_common_file_io_utils.py

```diff
@@ -191,7 +191,7 @@ async def _creator(num_upload_links: int, chunk_size: ByteSize) -> FileUploadSch
 @pytest.mark.skip(reason="this will allow to reproduce an issue")
 @pytest.mark.parametrize(
     "file_size,used_chunk_size",
-    [(parse_obj_as(ByteSize, "8.78Gib"), parse_obj_as(ByteSize, "10Mib"))],
+    [(parse_obj_as(ByteSize, 21800510238), parse_obj_as(ByteSize, 10485760))],
```

Comment:
why ``21800510238`` bytes vs the ``Gib`` equivalent?
why 2 instead of 8?

Reply:
because this is the issue that I found in Graylog. And I do not want to mess up the number. anyway that test is only meant for manual testing